### DISCUSSION TITLE
AI 判斷早期癌細胞

### DISCUSSION THREAD

**Discussion by manipulateed** [1101-GS4719-C/Python-Community-GS4719-C#921] [General]
這篇文章帶我看到人的靈感可以從一個在平凡不過的地方去研發出對人類非常有貢獻的工具，某些事情結合人工智慧，將能事半功倍

https://news.tvbs.com.tw/world/1995531

**Comment by ruthchen1020** [comment]
我覺得AI很快能夠取代人類的知識，但卻不能取代人類的智慧。
過去需要累積經驗去學習的東西，AI都可以很快的達到優良的水準，但卻唯獨在藝術圈裡充滿爭議，因為在藝術中，AI只是將人類的作品進行融合，就像人類作品的二創一樣。

**Comment by Zoe06060606** [comment]
AI的技術對未來醫療會有很大的幫助 希望對未來醫療能有一大進展

**Comment by peggy0329** [comment]
原來在醫療領域上，現在才有辨識腫瘤的系統，我原本以為AI辨識已經逐漸普及化～
不過這篇文章告訴我們也是因為有人類的「洞察」，才可以將AI有所應用
我覺得培養創新能力是我們現在人們需要重視的，才能合適的將AI技術化作工具

**Comment by Evan0979** [comment]
超級有用!普及之後肯定可以造福很多人!

**Comment by StevenWanglolz** [comment]
還記得之前看過影片討論AI醫生和真人醫生的比較，我忘記最後結論是甚麼了哈哈，不過AI在醫學的領域的應用還真是廣泛啊!

**Comment by CPCpyt** [comment]
我認為AI能夠彌補人在知識儲存的不足，
透過人辨認當下情況的知識，能夠帶來巨幅的改變。

**Comment by swerfg03joe** [comment]
AI在醫學上的應用多半是聚焦於、影像分析、和波形判斷，因為訓練AI時可以將很多不同病症丟進去一起訓練，所以只要給AI一張圖他就會告訴我們大概哪裡有問題，但這只是輔助診斷而已，還需要再進行後續的生化分析才能確認是何種病症。

**Comment by ru0372572** [comment]
AI的知識應用可不可行感覺要看是哪個領域耶，目前可能有些已經發展成熟，某些還在初步的階段，但一定都會越做越精細，越做越準確。

**Comment by BlahFish1006** [comment]
AI辨識可以有效協助醫師做診斷，尤其是當不同醫醫師對一個病人的診斷有分歧的時候，
它可以成為有利的客觀第三方「參考」，但實際上還是需要專業醫師來作最後的判斷和診治。

**Comment by harrylee1971** [comment]
很多醫療軟體現在都在專工AI這塊，逐漸地要把各大醫生給取代掉，到底是人工智慧還是是工人智慧呢?
